packages/test/src/test-interface-type-safety.ts

```diff
@@ -1,6 +1,7 @@
 import test from 'ava';
 import { defineSignal, defineQuery, ExternalWorkflowHandle, ChildWorkflowHandle, Workflow } from '@temporalio/workflow';
 import { WorkflowHandle } from '@temporalio/client';
+import * as wf from '@temporalio/workflow';
```

Comment:
I'd expect you to get `IllegalStateException`s while running your new tests, since `setHandle()` assert that there is an active workflow execution context. You may do this instead:

```
const wf = { setHandler: () => {} } as unknown as typeof import('@temporalio/workflow');
```

Reply:
Or move code that use `wf.setHandler()` inside an inner function that never gets called (like we did on `Can call signal on any WorkflowHandle`).